source/reference/minio-mc/mc-support-logs-enable.rst

```diff
@@ -0,0 +1,51 @@
+==========================
+``mc support logs enable``
+==========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support logs enable
+
+Description
+-----------
+
+Use the :mc-cmd:`mc support logs enable` command to allow real-time MinIO logs to upload to |subnet|.
```

Comment:
Should note this requires SUBNET registration (somewhere), is disabled by default, and is opt-in only (can be disabled at any time)

Should also check w/ engineering what logs (all of them? redacted? etc.) . Some companies have really strict controls over log upload as they can include PII or secure info

Reply:
Per @anjalshireesh , data is not scrubbed by default. Adding a note to that effect and suggesting to start the server with the `--anonymous` flag to hide sensitive information.